libs/wingc/src/ast.rs

```diff
@@ -418,16 +418,19 @@ pub struct Interface {
 	pub extends: Vec<UserDefinedType>,
 }
 
+#[derive(Debug)]
+pub enum BringSource {
+	BuiltinModule(Symbol),
+	JsiiModule(Symbol),
+	WingFile(Symbol),
+}
+
 #[derive(Debug)]
 pub enum StmtKind {
 	Bring {
-		module_name: Symbol, // Reference?
+		source: BringSource,
 		identifier: Option<Symbol>,
 	},
-	Module {
-		name: Symbol,
-		statements: Scope,
-	},
```

Comment:
I think there's merit to adding a syntax for creating modules within files, e.g.

```js
pub module Foo {
  pub class Bar {}
  pub class Baz {}
}
```

But that's not really the goal of this PR / I wanted to keep this PR on the smaller side, so I decided to remove it for now since the code would otherwise be unused.